client/auth/login.jsx

```diff
@@ -109,7 +109,7 @@ export const Login = createReactClass( {
 								<FormTextInput
 									name="login"
 									disabled={ requires2fa || inProgress }
-									placeholder={ translate( 'Username or email address' ) }
+									placeholder={ translate( 'Email address or username' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **69** times:
`translate( 'Username or Email Address' )` *ES Score: 13*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/login-flow)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).